src/app/controls/controls.html

```diff
@@ -1,202 +1,253 @@
+				     style="max-height:50px;">
+				<br>
+				<b>Crash NST</b> <br>
+				<h6>DepCow<br>CaneOfPacci</h6>
+			</div>
+			<div>
+				<img [ngClass]="{ 'active-background': vm.background == 'sms' }" (click)="setBackground('sms')" src="../../assets/sms/smsbgstream.png"
+				     style="max-height:50px;">
+				<br>
+				<b>Super Mario Sunshine</b> <br>
+				<h6>TonesBalones<br>SniperKing19</h6>
+			</div>
+			<div>
+				<img [ngClass]="{ 'active-background': vm.background == 'ootrando' }" (click)="setBackground('ootrando')" src="../../assets/ootrando/ootrandobgstream.png"
+				     style="max-height:50px;">
+				<br>
+				<b>OoT Rando</b> <br>
+				<h6>WoodenBarrel<br>Sonicshadowsilver2</h6>
+			</div>
+			<div>
+				<img [ngClass]="{ 'active-background': vm.background == 'uya' }" (click)="setBackground('uya')" src="../../assets/uya/uyabgstream.png"
+				     style="max-height:50px;">
+				<br>
+				<b>UYA</b> <br>
+				<h6>Ninjaratchet<br>CapnWiffle</h6>
+			</div>
+			<div>
+				<img [ngClass]="{ 'active-background': vm.background == 'bk' }" (click)="setBackground('bk')" src="../../assets/bk/bkbgstream.png"
+				     style="max-height:50px;">
+				<br>
+				<b>Banjo-Kazooie</b> <br>
+				<h6>Dickhiskhan<br>Hagginater</h6>
+			</div>
+			<div>
+				<img [ngClass]="{ 'active-background': vm.background == 'sa2b' }" (click)="setBackground('sa2b')" src="../../assets/sa2b/sa2bbgstream.png"
+				     style="max-height:50px;">
+				<br>
+				<b>Sonic Adventure 2</b> <br>
+				<h6>Talon2461<br>a_moustache</h6>
+			</div>
+			<div>
+				<img [ngClass]="{ 'active-background': vm.background == 'mmx' }" (click)="setBackground('mmx')" src="../../assets/mmx/mmxbgstream.png"
+				     style="max-height:50px;">
+				<br>
+				<b>Mega Man X</b> <br>
+				<h6>Walrus_Prime<br>Tokyo90</h6>
+			</div>
+			<div>
+				<img [ngClass]="{ 'active-background': vm.background == 'sm64' }" (click)="setBackground('sm64')" src="../../assets/sm64/sm64bgstream.png"
+				     style="max-height:50px;">
+				<br>
+				<b>Super Mario 64</b> <br>
+				<h6>BlueBob<br>HeisenburgerTV</h6>
+			</div>
+			<div>
+				<img [ngClass]="{ 'active-background': vm.background == 'srt' }" (click)="setBackground('srt')" src="../../assets/srt/srtbgstream.png"
+				     style="max-height:50px;">
+				<br>
+				<b>Spyro Reignited Trilogy</b> <br>
+				<h6>ChrisLBC<br>BubblesDelFuego</h6>
+			</div>
+			<div>
+				<img [ngClass]="{ 'active-background': vm.background == 'plge' }" (click)="setBackground('plge')" src="../../assets/plge/plgebgstream.png"
+				     style="max-height:50px;">
+				<br>
+				<b>Pokémon: Let's Go, Eevee!</b> <br>
+				<h6>Ouro<br>Keizaron</h6>
+			</div>
+			<div>
+				<img [ngClass]="{ 'active-background': vm.background == 'kh3' }" (click)="setBackground('kh3')" src="../../assets/kh3/kh3bgstream.png"
+				     style="max-height:50px;">
+				<br>
+				<b>KH3</b><br>
+				<h6>Ninten866<br>SwiftShadow</h6>
+			</div>
+			-->
```

Comment:
Just get rid of all the old ones but leave the assets in the assets folder. I want to make this more general anyways, so there's no reason to leave them commented out.